packages/server/graphql/mutations/helpers/updateQualAIMeetingsCount.ts

```diff
@@ -6,13 +7,20 @@ const updateQualAIMeetingsCount = async (
   teamId: string,
   dataLoader: DataLoaderWorker
 ) => {
-  const [meetingMembers, team, reflections, meeting] = await Promise.all([
+  const [meetingMembers, team, reflections, meeting, responses] = await Promise.all([
     dataLoader.get('meetingMembersByMeetingId').load(meetingId),
     dataLoader.get('teams').load(teamId),
     dataLoader.get('retroReflectionsByMeetingId').load(meetingId),
-    dataLoader.get('newMeetings').load(meetingId)
+    dataLoader.get('newMeetings').load(meetingId),
+    getTeamPromptResponsesByMeetingId(meetingId)
   ])
-  if (meetingMembers.length < 3 || !team || !meeting.summary || reflections.length < 5) return
+  if (
+    meetingMembers.length < 3 ||
+    !team ||
+    !meeting.summary ||
+    !(reflections.length >= 5 || responses.length >= 3)
```

Comment:
+1 The L&E mission is to increase Standups usage, so I wonder whether it'd be worth keeping the AI Summaries free for all users in Standups until you're happy with usage? 

The current implementation might help Self-Serve with conversions, so I won't advocate for my change too much 😄  but just an idea.

If you do want to keep it free, you'd need to remove the `canAccessAISummary` check, which has `return qualAIMeetingsCount < Threshold.MAX_QUAL_AI_MEETINGS`.

Reply:
L&E's focus this trimester is actually on signups, so I don't see a clear reason to make this one way or the other w.r.t. L&E's goals.

However, this makes me think that this would interfere with the trial-like experience for retro summaries. I.e. if a team holds daily recurring standups, they'd exceed the `MAX_QUAL_AI_MEETINGS` threshold within a few days.

I'll change this so that standup summaries are free for now.